### PR TITLE
Adding swagger api description about user and auth operations

### PR DESCRIPTION
**Pull request by vasap87**

### CODE REVIEW

File: doc/swagger-api-description/api.yaml [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+          description: Authorization user credential.
+          required: true
+          schema:
+            $ref: "#/definitions/Credential"
+      responses:
+        '201':
+          description: User session.
+          schema:
+            $ref: '#/definitions/Session'
+        '404':
```
Comment (kosmoflyko) [inline doc/swagger-api-description/api.yaml]: If user use not correct credentials server return 404 code too? :)

File: doc/swagger-api-description/api.yaml [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+      operationId: logoutSession
+      parameters:
+        - name: uuid
+          in: path
+          description: UUID of user session.
+          required: true
+          type: string
+      responses:
+        '204':
+          description: User updated.
```
Comment (kosmoflyko) [inline doc/swagger-api-description/api.yaml]: May be description should be "Session closed."

File: doc/swagger-api-description/api.yaml [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+        default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+
+definitions:
+
+  User:
+    type: object
+    description: Credentials of service users.
```
Comment (kosmoflyko) [inline doc/swagger-api-description/api.yaml]: Change description. This is not credentials, this is User object.

File: doc/swagger-api-description/api.yaml (hunk 1 above)
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: I will change it to 409/

File: doc/swagger-api-description/api.yaml [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+consumes:
+  - application/json; charset=utf-8
+
+produces:
+  - application/json; charset=utf-8
+
+basePath: /v1
+paths:
+  #See (Using HTTP Methods for RESTful Services)[http://www.restapitutorial.com/lessons/httpmethods.html]
```
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: Done.

File: doc/swagger-api-description/api.yaml [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+        '404':
+          description: Can not create new user.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+  #Change user password.
+    put:
```
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: Done.

File: doc/swagger-api-description/api.yaml [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+      '404':
+          description: User not found.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+      default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+  #Delete user.
+    delete:
```
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: Done with change parametr from user to uuid

File: doc/swagger-api-description/api.yaml [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+		'404':
+		  description: User not found on server.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+        default:
+          description: Unexpected error.
+          schema:
+            $ref: '#/definitions/ErrorModel'
+
+  /authorization:
```
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: Done.

File: doc/swagger-api-description/api.yaml [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+  #Create new user session.
+    post:
+      description: Creating new user session.
+      operationId: authorizationUser
```
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: Done.

File: doc/swagger-api-description/api.yaml (hunk 2 above)
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: Done.

File: doc/swagger-api-description/api.yaml (hunk 3 above)
Comment (vasap87) [inline doc/swagger-api-description/api.yaml]: I have changed it to "User of service".

**Pull request merged**